nri/css/style.css

```
@@ -0,0 +1,367 @@
+@charset "UTF-8";
+
+/* 共通部分
+-------------------- */
+
+html {
+    font-size: 100%;
+    font-feature-settings: "palt"
+}
+body{
+    font-family: 'Noto Sans JP',sans-serif!important;
```

`!important`の利用はやめましょう。（後でスタイルを書き換えることが難しくなるため）。
どうしても（外部のライブラリが利用していて、importantつけないと書き換えれない、など）のときだけにしましょう。

```suggestion
    font-family: 'Noto Sans JP',sans-serif;
```